### ISSUE TITLE
Bibliography for page

### ISSUE THREAD

**Issue by lpawela** [ali-ramadhan/DocumenterCitations.jl#26]
I would like to have a bibliography for a page with the references used only on that page. For instance here https://lpawela.github.io/numericalshadow.org/dev/numerical-shadow/ only few references are used (currently wrongly displayed due to #24 ) and I would like only these works displayed in the References section.

**Comment by charleskawczynski** [comment]
Just to note, there is a simple way to handle per-page citations in the documentation markdown. Here's an [example](https://juliaibpm.github.io/ViscousFlow.jl/latest/#References-1). That said, this approach is rather simple and doesn't support some nice things like cross-referencing.

**Comment by lpawela** [comment]
Yes, I can always try to parse the files, get all the citations, use `Bibliography.jl` to generate a proper bibliography for every page. Any other option is impossible in my use case - we have hundreds of citations, as our "documentation" is in fact a page that gathers all the knowledge on a specific subject.

**Comment by ali-ramadhan** [comment]
@lpawela Do you want to use one huge `.bib` file and put a bibliography on a page that only lists the references cited on that pages?

Or do you want to have multiple `.bib` files and the bibliography on each page lists all the references for some specific `.bib` file?

I guess the first one sounds more useful to you?

**Comment by lpawela** [comment]
The first one - one huge `.bib` with relevant citations put on the specific page

**Comment by Azzaare** [comment]
@lpawela sorry this is probably a bit late, but now Bibliography.jl supports selection of a subset of a bibliography.
This can probably be used easily in DocumenterCitations.jl to solve your problem.

I will go through as many issues as possible in the Bibliography.jl ecosystem this summer, so feel free to request other things.

For the reference, syntax is here: https://github.com/Humans-of-Julia/Bibliography.jl#short-documentation

**Comment by goerz** [comment]
This is now supported in the newly released [DocumenterCitations.jl v1.0](https://github.com/JuliaDocs/DocumenterCitations.jl/releases/tag/v1.0.0), see https://juliadocs.org/DocumenterCitations.jl/dev/syntax/#noncanonical
